src/translations/main_tr.ts

```diff
@@ -0,0 +1,822 @@
+        <source>Error starting Thunderbird, because we could not attach to the updater:
+
+%1</source>
+        <translation>Thunderbird başlatılırken hata oluştu, çünkü güncelleyiciye ekleyemedik:
+
+%1</translation>
+    </message>
+    <message>
+        <source>Ignore unread emails (now %1)</source>
+        <translation>Okunmamış e-postaları yoksay (şimdi%1)</translation>
+    </message>
+    <message>
+        <source>Error starting Thunderbird as &apos;%1 %2&apos;:
+
+%3</source>
+        <translation>Thunderbird &apos;%1 %2&apos; olarak başlatılırken hata oluştu:
+
+%3</translation>
+    </message>
+    <message>
+        <source>Warning: %1</source>
+        <translation>Uyarı: %1</translation>
+    </message>
+    <message>
+        <source>Don&apos;t ask again</source>
+        <translation>Bir daha sorma</translation>
+    </message>
+</context>
+<context>
+    <name>UnreadMonitor</name>
+    <message>
+        <source>Unable to watch %1 for changes.</source>
+        <translation>Değişiklikler için %1 izlenemiyor.</translation>
+    </message>
+    <message>
+        <source>Unable to read from %1.</source>
+        <translation>%1 okunamıyor.</translation>
+    </message>
+</context>
+<context>
+    <name>UpdateDialog</name>
+    <message>
+        <source>Birdtray Update</source>
+        <translation>Birdtray Güncellemesi</translation>
+    </message>
+    <message>
+        <source>Current version:</source>
+        <translation>Şimdiki sürüm:</translation>
+    </message>
+    <message>
+        <source>A new version of Birdtray is available.</source>
+        <translation>Birdtray&apos;in yeni bir sürümü mevcut.</translation>
+    </message>
+    <message>
+        <source>New version:</source>
+        <translation>Yeni sürüm:</translation>
+    </message>
+    <message>
+        <source>Download size:</source>
+        <translation>İndirme boyutu:</translation>
+    </message>
+    <message>
+        <source>Download</source>
+        <translation>İndir</translation>
+    </message>
+    <message>
+        <source>Ignore this version</source>
+        <translation>Bu sürümü yok say</translation>
+    </message>
+    <message>
+        <source>Update and Restart</source>
+        <translation>Güncelle ve Yeniden Başlat</translation>
+    </message>
+    <message>
+        <source>ca. %1 Mb</source>
+        <translation>%1 Mb</translation>
+    </message>
+</context>
+<context>
+    <name>UpdateDownloadDialog</name>
+    <message>
+        <source>Birdtray Update</source>
+        <translation>Birdtray Güncellemesi</translation>
+    </message>
+    <message>
+        <source>Downloading Birdtray update...</source>
+        <translation>Birdtray güncellemesi indiriliyor...</translation>
+    </message>
+    <message>
+        <source>Background</source>
+        <translation>Arka plân</translation>
+    </message>
+    <message>
+        <source>Downloading Birdtray installer...</source>
+        <translation></translation>
```

Comment:
```suggestion
        <translation type="unfinished"></translation>
```
Did you want to translate this text? If not, please set it to ` unfinished`.